src/buffers/memory_pool.c

```diff
@@ -8,4 +8,4 @@
  * Memory pool variable prevents code segments flowing into the memory pool
  * for easier portability.
 */
-u8 memoryPool[MEMORY_POOL_SIZE];
+u8 gMemoryPool[MEMORY_POOL_SIZE];
```

Comment:
Fair.

But this variable should never actually be used. If anything it should be a static const if that's even possible.

Putting g makes it seem like it could be used. Sould be sMemoryPool at the least.

Or perhaps even sMemoryPoolBss or something (if this is bss)

Reply:
I rename to sMemoryPool and add a warning
![image](https://github.com/user-attachments/assets/b8af35b7-07c1-4733-8b1c-0f8248d2ed5f)